lib/src/components/mixins/collidable.dart

```diff
@@ -4,6 +4,11 @@ import '../../geometry/rectangle.dart';
 import 'hitbox.dart';
 
 mixin Collidable on Hitbox {
+  /// Set whether [collidable] is active
+  /// If [active collision] is set to true, collision with other [collisionable] will be actively checked in [collisiondetection]
```

Comment:
The words inside of [ ] should correspond to what they are called in the code.

Reply:
OK I'll make the changes right away